### PR TITLE
LOG-3149: Update operator-sdk and k8s dependencies

### PR DESCRIPTION
**Pull request by xperimental**
### Description

Updates operator-sdk to 1.24.0 and Kubernetes client code to 1.25.2

### Links

- JIRA: [LOG-3149](https://issues.redhat.com//browse/LOG-3149)

### CODE REVIEW

File: .bingo/kustomize.mod [hunk 1]
```diff
@@ -2,4 +2,4 @@ module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
 go 1.18
 
-require sigs.k8s.io/kustomize/kustomize/v3 v3.8.7
+require sigs.k8s.io/kustomize/kustomize/v4 v4.5.5
```
Comment (Red-GV) [inline .bingo/kustomize.mod:5]: I thought that `kustomize`'s max version is 4.5.7?

**Review by Red-GV — COMMENTED** [review]
Just a comment here. Otherwise, looks fine to me.

File: .bingo/kustomize.mod (hunk 1 above)
Comment (xperimental) [reply .bingo/kustomize.mod]: I just used 4.5.5 because it was specifically mentioned in the operator-sdk upgrade instructions. Updated to 4.5.7 now :+1:

**Pull request merged**